open-hackathon/src/hackathon/views.py

```diff
@@ -220,3 +224,18 @@ def get(self):
 api.add_resource(UserResource, "/api/user")
 api.add_resource(CurrentTime, "/api/currenttime")
 
+# ------------------------------ APIs for admin-site --------------------------------
+class AdminHackathonsResource(Resource):
+    @admin_token_required
+    def get(self):
+        admin = g.admin
+        return admin_manager.get_hackid_from_adminID(admin.id)
+
+
+    def post(self):
+        return ""
+
+
+
+
```

Comment:
e……i just want to separate the ‘classResource’  and the ‘api.add_resource’ to remind others to add ClassResource in these blanks ； do not add classResource after the ‘api.add_resource’

maybe it is a bad habit in src code layout ，and i will take care of this